### DISCUSSION TITLE
Access file from import

### DISCUSSION THREAD

**Discussion by f-rozanski** [LemonAppDev/konsist#1216] [Request a Feature]
### Tell us more.

It will be convenient to access `KoFileDeclaration` of a given import 🙏  I have a use case with a multi-module backend application when I want to write rules on which files/classes could be imported between modules.

example:
```
val importsFromAnotherModule = Konsist.scopeFromModule("module-a")
            .imports
            .filter { it.sourceFile.moduleName != "module-a"}
```

**Comment by igorwojda** [comment]
Now import has `containingFile` property
